src/Text/Pandoc/Options.hs

```diff
@@ -363,6 +363,7 @@ data WriterOptions = WriterOptions
   , writerBeamer           :: Bool       -- ^ Produce beamer LaTeX slide show
   , writerSlideLevel       :: Maybe Int  -- ^ Force header level of slides
   , writerChapters         :: Bool       -- ^ Use "chapter" for top-level sects
+  , writerParts            :: Bool       -- ^ Use "part" for top-level sects in LaTeX
```

Comment:
> I don't really like having two independent booleans for chapters and parts.

I was afraid you'd say that.  I gave it a shot and added `--top-level-division=[section|chapter|parts]`, deprecating `--chapters`.  The new MANUAL section probably needs the attention of a native speaker.